jets-bench/benches/elements/main.rs

```diff
@@ -125,14 +92,28 @@ impl ElementsBenchEnvType {
     }
 }
 
+fn jet_arrow(jet: Elements, pows_of_two: &[Type]) -> (Arc<types::Final>, Arc<types::Final>) {
+    let src_ty = jet
+        .source_ty()
+        .to_type(|n| pows_of_two[n].shallow_clone())
```

Comment:
nit: This can use the `to_bit_width` from #110. But this is only used in benchmarks, not necessary to change this.